packages/template/vite-typescript/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@electron-forge/template-vite-typescript",
+  "version": "6.3.0",
+  "description": "Vite-TypeScript template for Electron Forge, gets you started with Vite really quickly",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/electron/forge",
+    "directory": "packages/template/vite-typescript"
+  },
+  "author": "caoxiemeihao",
+  "license": "MIT",
+  "main": "dist/ViteTypeScriptTemplate.js",
```

Comment:
Can this be updated with a `"files"` field, like #3284 added to all packages?

Reply:
Okay! done.

Reply:
Thanks! 😄